src/main/java/dev/dokan/dokan_java/masking/MaskValueSetImpl.java

```diff
@@ -31,6 +51,17 @@ public final void add(T... items) {
         Arrays.stream(items).filter(Objects::nonNull).forEach(this::add);
     }
 
+    @Override
+    public EnumSet<T> elements() {
+        return this.elements.clone();
```

Comment:
I don't like this method. It says "elements", but rather than just give a view on the elements it returns a copy of them. So either it should be renamed or removed.

Reply:
Giving back a copy is a common approach to this problem tho. Editing MaskValueSet using the underlying EnumSet is not useful, but getting a copy can be.

Reply:
Point taken. But maybe this will be obsolete, depeding on the decision to keep this method in the implemented interface.

Reply:
Ah, ok. I didn't knew about that... But now it is already changed^^

Reply:
Point taken. But maybe this will be obsolete, depeding on the decision to keep this method in the implemented interface.